.evergreen/config.yml

```diff
@@ -487,6 +487,10 @@ functions:
             export SINGLE_MONGOS_LB_URI="${MONGODB_URI}"
             export MULTI_MONGOS_LB_URI="${MONGODB_URI}"
           fi
+          if [ -n "${TEST_INDEX_MANAGEMENT}" ]; then
+            export DB_USER=${DRIVERS_ATLAS_LAMBDA_USER}
+            export DB_PASSWORD=${DRIVERS_ATLAS_LAMBDA_PASSWORD}
+          fi
 
           MONGODB_VERSION=${VERSION} \
```

Comment:
The index tests are [never run even in the test-search-index-helpers task](https://spruce.mongodb.com/task/mongo_python_driver_test_search_index_helpers__platform~rhel8_python_version~3.8_test_search_index_helpers_patch_b7796e17948a5c7151e51c4ae5348bd485609876_64ca7aeb3627e055428565f5_23_08_02_15_48_59/logs?execution=0):
```
 [2023/08/02 16:08:01.192]   test_standalone_error (test_heartbeat_monitoring.TestHeartbeatMonitoring) ... ok (0.102s)
 [2023/08/02 16:08:01.194]   test_inputs (test_index_management.TestCreateSearchIndex) ... skip (0.001s)
 [2023/08/02 16:08:01.196]   test_basic (test_json_util.TestJsonUtil) ... ok (0.001s)
```

We need to forward TEST_INDEX_MANAGEMENT here.